pkg/scheduler/framework/types.go

```diff
@@ -337,6 +337,9 @@ type Diagnosis struct {
 	PreFilterMsg string
 	// PostFilterMsg records the messages returned from PostFilter plugins.
 	PostFilterMsg string
+	// EvaluatedNodes records the number of nodes reached Filter stage.
```

Comment:
nit: Let's make it more accurate: 

```suggestion
	// EvaluatedNodes records the number of nodes evaluated by Filter stage.
```

(even if Pods reach Filter stage, that is, went through PreFilterResult(s), not all of them are evaluated by Filter Plugins because of `numNodesToFind` (pkg/scheduler/schedule_one.go#584))